packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -193,21 +193,15 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
     context: Context,
     did: string
   ): Promise<DIDResolutionResult> {
-    let result
     try {
-      result = await context.did.verifyJWS(commit, {
+      const verification = await context.did.verifyJWS(commit, {
         atTime: meta.timestamp,
+        issuer: did,
         disableTimecheck: meta.disableTimecheck,
       })
+      return verification.didResolutionResult
     } catch (e) {
       throw new Error('Invalid signature for signed commit. ' + e)
     }
```

Comment:
Why is this wrapped in a try catch and an error is rethrown? Seems better to just let the error from *verifyJWS* propagate?

Reply:
Was here before. Decided not to change this.

Reply:
Changed.